.gitignore

```diff
@@ -0,0 +1,114 @@
+# container ids
+*.running_container_id*
+
+# Created by https://www.gitignore.io/api/python,pycharm
```

Comment:
I think MIT as the content of the generated gitignores comes from github/gitignore .

Gitignore.io itself uses GPL I think: https://github.com/joeblau/gitignore.io#license